### PR TITLE
test: add Ethereum sign in e2e

### PR DESCRIPTION
**Pull request by C-Ivan**
## **Description**

This PR is within the scope of the Quality Quest.

We're including an e2e test case to check the sign in with Ethereum feature.

**Test steps:**
Given I am on the test dapp
When I tap on the Sign in with ethereum button
Then the Signature form should appear
When I tap cancel
Then the signature form should disappear
When I tap on the Sign in with ethereum button
And I tap Sign
Then the signature form should disappear
_And the signature result should not be empty_  ← This step is pending and will be done in the future.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [✓] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [✓] I've completed the PR template to the best of my ability
- [✓] I’ve included tests if applicable
- [✓] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [✓] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by C-Ivan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: e2e/selectors/Browser/TestDapp.selectors.js [hunk 1]
```diff
@@ -9,6 +9,6 @@ export const TestDappSelectorsWebIDs = {
   SIGN_TYPE_DATA: 'signTypedData',
   SIGN_TYPE_DATA_V3: 'signTypedDataV3',
   SIGN_TYPE_DATA_V4: 'signTypedDataV4',
-  SIWE: 'siwe',
+  ETHEREUM_SIGN: 'siwe',
```
Comment (C-Ivan) [inline e2e/selectors/Browser/TestDapp.selectors.js:12]: Renamed selector for better readability.

File: e2e/specs/confirmations/signatures/ethereum-sign.spec.js
Comment (C-Ivan) [inline e2e/specs/confirmations/signatures/ethereum-sign.spec.js:1]: New e2e case for Ethereum sign with test dApp, based on similar signing cases (personal and typed)

File: e2e/pages/Browser/TestDApp.js [hunk 2]
```diff
@@ -72,6 +72,12 @@ class TestDApp {
       TestDappSelectorsWebIDs.SIGN_TYPE_DATA_V4,
     );
   }
+  get ethereumSignButton() {
```
Comment (C-Ivan) [inline e2e/pages/Browser/TestDApp.js:75]: Selector object for Ethereum sign in button in test dApp

File: e2e/pages/Browser/TestDApp.js [hunk 3]
```diff
@@ -124,6 +130,11 @@ class TestDApp {
   async tapTypedV4SignButton() {
     await this.tapButton(this.signTypedDataV4Button);
   }
+
+  async tapEthereumSignButton() {
```
Comment (C-Ivan) [inline e2e/pages/Browser/TestDApp.js:134]: Tapping the Ethereum sign in button in test dApp

**Review by Andepande — APPROVED** [review]
LGTM

**Pull request merged**